File: frontend-biblio/app/src/main/java/com/fex/projetandroidm1/model/Pret.java
package com.fex.projetandroidm1.model;

import org.json.JSONException;
import org.json.JSONObject;

public class Pret {

    private String numlecteur;
    private String numlivre;
    private String datepret;

    public Pret() {
    }

    public Pret(String numlecteur, String numlivre, String datepret) {
        this.numlecteur = numlecteur;
        this.numlivre = numlivre;
        this.datepret = datepret;
    }

    public String getNumlecteur() {
        return numlecteur;
    }

    public void setNumlecteur(String numlecteur) {
        this.numlecteur = numlecteur;
    }

    public String getNumlivre() {
        return numlivre;
    }

    public void setNumlivre(String numlivre) {
        this.numlivre = numlivre;
    }

    public String getDatepret() {
        return datepret;
    }

    public void setDatepret(String datepret) {
        this.datepret = datepret;
    }
}

File: backend-biblio/src/Entity/Pret.php
<?php

namespace App\Entity;

use ApiPlatform\Core\Annotation\ApiResource;
use App\Repository\PretRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ApiResource(formats= {"json"})
 * @ORM\Entity(repositoryClass=PretRepository::class)
 */
class Pret
{
    /**
     * @ORM\Id
     * @ORM\Column(type="string", length=25)
     */
    private $numlecteur;
    /**
     * @ORM\Id
     * @ORM\Column(type="string", length=25)
     */
    private $numlivre;
    /**
     * @ORM\Column(type="string", length=25)
     */
    private $datepret;

    public function getNumlecteur(): ?string
    {
        return $this->numlecteur;
    }
    public function setNumlecteur($numlecteur)
    {
        $this->numlecteur = $numlecteur;
    }
    public function getNumlivre(): ?string
    {
        return $this->numlivre;
    }
    public function setNumlivre($numlivre)
    {
        $this->numlivre = $numlivre;
    }
    public function getDatepret(): ?string
    {
        return $this->datepret;
    }
    public function setDatepret($datepret)
    {
        $this->datepret = $datepret;
    }

    //Parameters : numlecteur=1;numlivre=L1
}
